walkthroughs/3a-low-code-api-development/walkthrough.adoc

```diff
@@ -264,29 +265,29 @@ Fuse Online automatically completes the type *String as String*.
 
 
 . Click *Save* in the top right corner of the page to save your work.
-The *New Integration* wizard is displayed.
+The *Review Actions* screen of the *New Integration* wizard is displayed.
 
 . Click *Next*.
 
-. Click *Publish* on the *New Integration* wizard screen.
+. Click *Save* on the *New Integration* wizard screen.
 
-. When prompted to *Give this integration a name*:
-.. Enter the following in the *Integration Name* field:
+. On the *Save the integration* screen:
+.. Enter the following in the *Name* field:
 +
 ----
-Greeting Integration
+Greeting Integration {user-sanitized-username}
```

Comment:
Making this change so each user can identify their application in the single 3scale tenant we use.

Reply:
This does not seem to render correctly:

![greeting](https://user-images.githubusercontent.com/1851198/70335680-0c391180-1848-11ea-81b2-70a19e26c994.png)